tests/conftest.py

```diff
@@ -1313,6 +1318,15 @@ def finalizer():
 
 @pytest.fixture()
 def mcg_obj(request):
+    return mcg_obj_fixture(request)
+
+
+@pytest.fixture(scope='session')
+def mcg_obj_session(request):
```

Comment:
First, it is a good idea to instantiate MCG per session. This is described in https://github.com/red-hat-storage/ocs-ci/issues/1178

I'm not though if with the current MCG cluster members, we can instantiate a single MCG for the entire session, as some of them are specific for a region / cloud provider.

Reply:
I need to instantiate it to session or class as upgrade test cases are broken into more tests (before and after upgrade).

Reply:
Commented on the issue and described the situation - I think it's a subject for another PR